doc/release/0.5.rst

```diff
@@ -9,6 +9,9 @@ Release Date: TBD
 New Features
 ------------
 #. Move instead of copy files from TemporaryDirectoryChanger. (`PR#2022 <https://github.com/terrapower/armi/pull/2022>`_)
+#. Provide ``Composite.iterChildren``, ``Composite.iterChildrenWithFlags``, ``Composite.iterChildrenOfType``,
```

Comment:
ARMI 0.5.0 was released.

Can we move this up in this file to the 0.5.1 release notes? 

Thanks!

Reply:
This is in the 0.5.1 section